scripts/generate-tmlanguage.mts

```diff
@@ -12,7 +12,7 @@
 import * as fs from 'fs/promises';
 import * as tokens from '../packages/language/src/parser/tokens.js';
 
-const keywords = Array.from(tokens.keywordMap, ([key]) => key.toLowerCase()).toSorted();
+const keywords = Array.from(tokens.keywordMap, ([key]) => key.toLowerCase()).sort();
```

Comment:
toSorted() is supported by ES2023+ (which doesn't seem to be used).

Difference: sort() mutates the array in place but Array.from() does already create a new array instance.